### ISSUE TITLE
Training time on the Residue (RES) Dataset

### ISSUE THREAD

**Issue by tuanle618** [drorlab/gvp-pytorch#7]
Hi Bowen,

thanks for sharing this nice code base of the GVP model in combination with the Atom3D benchmark.
I am currently running the Residue https://www.atom3d.ai/res.html benchmark and noticed that the dataset is quite large with 3,733,710 samples in the training set (See Appendix D.3 in https://arxiv.org/pdf/2012.04035.pdf) - I was wondering how long you trained the GVP-GNN on the RES dataset? In the default arguments, the maximum training time per epoch is set to 120 minutes.https://github.com/drorlab/gvp-pytorch/blob/82af6b22eaf8311c15733117b0071408d24ed877/run_atom3d.py#L16-L17 

I am currently training a similar model to GVP using batch-size 32 on the RES dataset and require ~5.5hours on an NVIDIA V100 GPU per epoch. Could you tell me, how long you trained your model to obtain 0.527 ± 0.003 test accuracy?

Thanks!
